File: src/main/java/com/sudosoo/takeiteasy/dto/heart/CommentHeartRequestDto.java
package com.sudosoo.takeiteasy.dto.heart;

import jakarta.validation.constraints.NotNull;
import lombok.*;

@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
public class CommentHeartRequestDto {
    @NotNull(message = "유저아이디를 입력해 주세요.")
    private Long memberId;
    @NotNull(message = "댓글아이디를 입력해 주세요.")
    private Long commentId;
}

File: takeItEasy-application/src/main/kotlin/com/sudosoo/takeItEasy/dto/heart/CommentHeartRequestDto.kt
package com.sudosoo.takeItEasy.dto.heart

import jakarta.validation.constraints.NotBlank

class CommentHeartRequestDto(
    @NotBlank(message = "멤버아이디를 입력해 주세요.")
    val memberId: Long,
    @NotBlank(message = "댓글아이디를 입력해 주세요.")
    val commentId: Long
)